test/integration/services/data/test-update-contracted-hours-for-workload-owner.js

```diff
@@ -0,0 +1,44 @@
+const expect = require('chai').expect
+
+const dataHelper = require('../../../helpers/data/aggregated-data-helper')
+const updateContractedHoursForWorkloadOwner = require('../../../../app/services/data/update-contracted-hours-for-workload-owner')
+const getContractedHoursForWorkloadOwner = require('../../../../app/services/data/get-contracted-hours-for-workload-owner')
+
+const UPDATEDHOURS = 22
+var inserts = []
+
+describe('services/data/update-contracted-hours-for-workload-owner', function () {
+  before(function () {
+    return dataHelper.addWorkloadCapacitiesForOffenderManager()
+      .then(function (builtInserts) {
+        inserts = builtInserts
+      })
+  })
+
+  it('should update the contracted hours for a workload owner and return count of updated rows', function () {
```

Comment:
Is a count of the updated rows what is returned as default?

Reply:
It sure is